### DISCUSSION TITLE
What is system requirement?

### DISCUSSION THREAD

**Discussion by A13xund3r** [alicevision/Meshroom#2069] [Q&A]
So far I was using version 2019 without issues, but latest 2023 is giving me corrupted image error if use program import menu regardless of file location or if file name is short or long. My PC is running windows 7, copied all data to win 10 machine and everything is working straight away.  Can't find any system requirement info.

**Comment by natowi** [comment]
Windows 7 has reached its end of life in January this Year. Newer versions of Meshroom are not compiled with Windows 7 in mind and are not officially supported. I don´t know if it is the the issue in your case, as I don´t have any Windows 7 machine, but some Meshroom dependencies may not be compatible with this old OS. Updating your computer to a newer Windows version or installing a Linux distribution is highly recommended in any case.
